usagov/vote-gov-drupal
# Vote-1457 -> visual regression for static site

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
Vote-1457

## Description (optional)
Pr is visual regression testing 

## Deployment and testing (required, if applicable)
### Testing steps
1. go into testing and open the `cypress-visual-report` and open that file in your browser you can see the report
